src/patternfly/components/FormControl/form-control.scss

```diff
@@ -263,198 +186,170 @@ $pf-v5-c-form-control--m-clock--Coordinates: "M256 8C119 8 8 119 8 256s111 248 2
     // stylelint-disable-next-line
     @-moz-document url-prefix() {
-      --#{$form-control}--PaddingRight: calc(var(--#{$form-control}__select--PaddingRight) - 1px);
-      --#{$form-control}--PaddingLeft: calc(var(--#{$form-control}__select--PaddingLeft) - 4px);
+      --#{$form-control}--PaddingRight: calc(var(--#{$form-control}--select--PaddingRight) - 1px);
+      --#{$form-control}--PaddingLeft: calc(var(--#{$form-control}--select--PaddingLeft) - 4px);
     }
 
-    &[aria-invalid="true"] {
-      --#{$form-control}--PaddingRight: var(--#{$form-control}__select--invalid--PaddingRight);
-      --#{$form-control}--invalid--BackgroundPosition: var(--#{$form-control}__select--invalid--BackgroundPosition); // drop this declaration and use the __select var directly in the background-position below in breaking change release
-
-      background-image: var(--#{$form-control}__select--BackgroundUrl), var(--#{$form-control}--invalid--BackgroundUrl);
-      background-position: var(--#{$form-control}__select--BackgroundPosition), var(--#{$form-control}--invalid--BackgroundPosition);
-      background-size: var(--#{$form-control}__select--BackgroundSize), var(--#{$form-control}--invalid--BackgroundSize);
+    .pf-m-error {
+      --#{$form-control}--PaddingRight: var(--#{$form-control}--select--invalid--PaddingRight);
     }
 
-    &.pf-m-success {
-      --#{$form-control}--PaddingRight: var(--#{$form-control}__select--success--PaddingRight);
-      --#{$form-control}--success--BackgroundPosition: var(--#{$form-control}__select--success--BackgroundPosition); // drop this declaration and use the __select var directly in the background-position below in breaking change release
-
-      background-image: var(--#{$form-control}__select--BackgroundUrl), var(--#{$form-control}--success--BackgroundUrl);
-      background-position: var(--#{$form-control}__select--BackgroundPosition), var(--#{$form-control}--success--BackgroundPosition);
-      background-size: var(--#{$form-control}__select--BackgroundSize), var(--#{$form-control}--success--BackgroundSize);
+    .pf-m-success {
+      --#{$form-control}--PaddingRight: var(--#{$form-control}--select--success--PaddingRight);
     }
 
-    &.pf-m-warning {
-      --#{$form-control}--PaddingRight: var(--#{$form-control}__select--m-warning--PaddingRight);
-
-      background-image: var(--#{$form-control}__select--BackgroundUrl), var(--#{$form-control}--m-warning--BackgroundUrl);
-      background-position: var(--#{$form-control}__select--BackgroundPosition), var(--#{$form-control}__select--m-warning--BackgroundPosition);
-      background-size: var(--#{$form-control}__select--BackgroundSize), var(--#{$form-control}--m-warning--BackgroundSize);
+    .pf-m-warning {
+      --#{$form-control}--PaddingRight: var(--#{$form-control}--select--m-warning--PaddingRight);
     }
+  }
 
-    &.pf-m-placeholder {
-      color: var(--#{$form-control}--placeholder--Color);
+  // TODO FIX THIS
```

Comment:
Oh yeah - I thought I had a bug there because once selected, the text stays the placeholder color, but that seems to be the way it is currently. Is that the intention? It seemed like this was trying to fix that but didn't.

Reply:
yep, we're making a faux-placeholder for the select component. You select the placeholder item and apply the placeholder class to the parent to make the text look grey, like it's just a placeholder. It makes more sense if you